Formula/python3.rb

```diff
@@ -47,17 +29,17 @@ class Python3 < Formula
   depends_on :x11 if build.with?("tcl-tk") && Tab.for_name("homebrew/dupes/tcl-tk").with?("x11")
   depends_on "sphinx-doc" => [:build, :optional]
 
-  skip_clean "bin/pip3", "bin/pip-3.4", "bin/pip-3.5"
-  skip_clean "bin/easy_install3", "bin/easy_install-3.4", "bin/easy_install-3.5"
+  skip_clean "bin/pip3", "bin/pip-3.4", "bin/pip-3.5", "bin/pip-3.6"
+  skip_clean "bin/easy_install3", "bin/easy_install-3.4", "bin/easy_install-3.5", "bin/easy_install-3.6"
 
   resource "setuptools" do
```

Comment:
I realize this won't be a popular opinion for people waiting for this PR to land, but I think setuptools should be upgraded to 32.2.0, or at least 32.1.0 to match the version in [`python.rb`](https://github.com/Homebrew/homebrew-core/blob/7330a126bf338b5216dc3b31bb65485cbedbbbf1/Formula/python.rb#L44-L47).